tests/functional/tests/test_local_users.py

```diff
@@ -25,7 +26,7 @@ def wait_for_application_states(self):
         zaza.model.block_until_all_applications_idle() seems to not wait for
         (config-changed) to settle. Causing a race.
         """
-        zaza.model.wait_for_agent_status()
```

Comment:
Why does `zaza.model.wait_for_agent_status()` not work anymore? I feel it(or any alternative) would be better than `sleep` if we can make it work.

Reply:
Depends on how fast the status hook fire.
It's working only because the hook fire slower than this line on our previous environment. If it fire before this line then it will stuck here forever.

Reply:
Alright, so it's not better than `sleep` in any way in this case? In that case LGTM.